core/src/main/resources/org/kohsuke/stapler/bind.js

```diff
@@ -26,39 +26,43 @@ function makeStaplerProxy(url,crumb,methods) {
             for (var i=0; i<args.length-(callback!=null?1:0); i++)
                 a.push(args[i]);
 
-            if(window.jQuery === window.$) { //Is jQuery the active framework?
-                $.ajax({
-                    type: "POST",
-                    url: url+methodName,
-                    data: stringify(a),
-                    contentType: 'application/x-stapler-method-invocation;charset=UTF-8',
-                    headers: {'Crumb':crumb},
-                    dataType: "json",
-                    success: function(data, textStatus, jqXHR) {
-                        if (callback!=null) {
+            var headers = {
+                'Content-Type': 'application/x-stapler-method-invocation;charset=UTF-8',
+                'Crumb': staplerCrumb,
+            };
+            // If running in Jenkins, add Jenkins-Crumb header.
+            if (typeof crumb !== 'undefined') {
+                headers = crumb.wrap(headers);
+            }
+
+            fetch(url + methodName, {
+                method: 'POST',
+                headers: headers,
+                body: stringify(a),
```

Comment:
Note: `JSON.stringify` _cannot_ be used here, because it blows up when Prototype is present (which is not the case in this repository's test suite, but _is_ the case in core's test suite and core's production environment). Fortunately Kohsuke did provide a custom `stringify` method for use with the dead jQuery code path, which we can use here and which solves the problem.